example/package.json

```diff
@@ -22,24 +22,22 @@
     "eslint-plugin-react": "^7.14.3",
     "get-dev-paths": "^0.1.1",
     "metro-config": "^0.56.0",
-    "react": "16.11.0",
-    "react-art": "^16.9.0",
-    "react-dom": "^16.9.0",
-    "react-native": "0.62.2",
-    "react-native-gesture-handler": "1.4.0",
+    "react": "17.0.1",
+    "react-native": "0.64.0",
+    "react-native-gesture-handler": "1.10.3",
     "react-native-web": "^0.11.7",
     "react-navigation": "^3.12.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.9.6",
-    "@babel/runtime": "^7.9.6",
+    "@babel/core": "^7.12.9",
+    "@babel/runtime": "^7.12.5",
     "babel-plugin-module-resolver": "^4.0.0",
-    "metro-react-native-babel-preset": "^0.59.0",
-    "babel-jest": "24.9.0",
-    "jest": "24.9.0",
+    "metro-react-native-babel-preset": "^0.64.0",
+    "babel-jest": "^26.6.3",
+    "jest": "^26.6.3",
     "prettier": "^1.18.2",
     "prop-types": "^15.7.2",
-    "react-native-screens": "^1.0.0-alpha.22",
+    "react-native-screens": "^2.18.1",
```

Comment:
I would like to remove this dependency at some point. I think I used it because I didn't know how to make React Native apps properly and I wanted to navigate between screens like native apps do. We can basically take Magic Weather and adapt it to our own needs.